spec/models/authenticator/ldap_spec.rb

```diff
@@ -163,6 +164,20 @@ def authenticate
     let(:username) { 'alice' }
     let(:password) { 'secret' }
 
+    context "when using LDAP" do
+      let(:config) { super().merge(:ldap_role => true) }
```

Comment:
Thanks @jvlcek, I had a hunch that `super()` was used to discard the args but it's not obvious why, maybe because the block arg is different... who knows.  :sparkles: 

Either way, I prefer @chrisarcand's latter example above, but yeah, it's fine for this PR.